source/meetings/trustees/2023-03-04.rst

```diff
@@ -23,3 +23,4 @@ Decisions
 ---------
 
 - :doc:`../../decisions/2023-03-04-invocation-of-clause-15.1.2`
+- :doc:`../../policies/trustee-ballot-policy`
```

Comment:
```suggestion
- Creation of :doc:`../../policies/trustee-ballot-policy`
```

Reply:
I know we dropped this after 2019 but I think it's nice in the Decisions list to differentiate between "here's a decision" and "here's a policy we decided" cf. https://ukpa-internaldocs.readthedocs.io/en/latest/meetings/trustees/2019-11-30/